File: rfmeask/tools/rs485test.rb
#!/usr/bin/ruby
#coding: utf-8

require "serialport"

port = SerialPort.new "/dev/ttyS3", 115200
port.parity= SerialPort::EVEN
port.data_bits= 8
port.stop_bits= 1

port.print "\001201\00200\003F8"

loop do
	p port.read 1
end


port.close

File: rfmeask/tools/rs485test.py
#!/usr/bin/env python3
#coding: utf-8

import serial

port = serial.Serial("/dev/ttyS3", 115200, timeout=1, parity=serial.PARITY_EVEN, bytesize=8, stopbits=1)

btes = "\001101\00200\003F7".encode("utf-8")
#btes = "\001201\00200\003F8".encode("utf-8")
#btes = "\001301\00200\003F9".encode("utf-8")

print(btes)
port.write(btes)

while True:
	print(port.read(1))

port.close()
